include/dns_records.hrl

```diff
@@ -236,6 +236,14 @@
     salt :: binary()
 }).
 
+%% TLSA record for DANE TLS. See RFC 6698.
+-record(dns_rrdata_tlsa, {
+    usage :: dns:uint8(),
+    selector :: dns:uint8(),
+    matching_type :: dns:uint8(),
+    certificate_association_data :: binary()
```

Comment:
While the RFC section refers to the "Certificate Association Data", I think we can probably get away with a simpler "certificate". Both are used often interchangeably in the docs, and in most implementations I've seen it's simply called certificate.

(similarly, certificate usage is called "usage" as you've done here).

Reply:
Thanks, I'll rename the field.